detectors/node/opentelemetry-resource-detector-aws/src/detectors/index.ts

```diff
@@ -15,7 +15,14 @@
  */
 
 export * from './AwsEc2Detector';
-export * from './AwsBeanstalkDetector';
+export {
```

Comment:
note for reviewer: small step forward for https://github.com/open-telemetry/opentelemetry-js/issues/4186